server/src/http.rs

```diff
@@ -195,10 +196,11 @@ impl<Q: QueryExecutor + 'static> Service<Q> {
             })
     }
 
+    // POST /debug/flush_memtable
```

Comment:
Maybe this url should be prefixed with `admin`.

Reply:
I think this interface is mainly used for debugging, not for operation purpose.